save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt

```diff
@@ -123,7 +124,12 @@ class DockerService(private val configProperties: ConfigProperties) {
             ClassPathResource(SAVE_CLI_EXECUTABLE_NAME).inputStream,
             File(resourcesPath, SAVE_CLI_EXECUTABLE_NAME)
         )
+        val baseImage = execution
+            .sdk
+            .split(";")
+            .joinToString("\n") { "FROM $it" }
```

Comment:
I don't think this works as you are expecting, if you want to reuse output of the previuos `FROM` instruction, you'll need [some additional steps](https://docs.docker.com/engine/reference/builder/#from)